robottelo/vm.py

```diff
@@ -672,18 +672,19 @@ def configure_puppet(self, rhel_repo=None, proxy_hostname=None):
             'report          = true\n'
             'ignoreschedules = true\n'
             'ca_server       = {0}\n'
+            'certname        = {1}\n'
             'environment     = production\n'
-            'server          = {1}\n'.format(proxy_hostname, proxy_hostname)
+            'server          = {0}\n'.format(proxy_hostname, self.hostname)
```

Comment:
You should now be able to use f'strings.

Reply:
@jyejare Is it ok if raise different pr for it? This file is huge and there are lot of replacement that would need to be done.

Reply:
You can, in fact, it's a Rottelo-wide change. For now, you may choose to fix it just for this PR. Leaving up to you.

Reply:
@jyejare fixed it just for my pr now. Will find some time to fix it for whole file as well.